functions/snippets/pom.xml

```diff
@@ -102,10 +114,36 @@
     </dependency>
   </dependencies>
 
-  <!-- Required for Java 8 (Alpha) functions in the inline editor -->
+  <!-- Disable tests when deploying to GCF -->
+  <!-- (where .gcloudignore isn't present) -->
+  <profiles>
+    <profile>
+      <activation>
+        <file>
+          <missing>.gcloudignore</missing>
+        </file>
+      </activation>
```

Comment:
.gcloudignore isn't going to github, so you are turning off our tests.

Reply:
Found an alternative - see b/151989174 for a proposed longer-term solution.